### PR TITLE
[PWA-1553] PB verify block

### PR DESCRIPTION
**Pull request by michaelyu0123**
## Description

To verify that the Page Builder Block content type renders correctly on Venia for each content type property.

## Related Issue

Closes [PWA-1553](https://jira.corp.magento.com/browse/PWA-1553): Verify Block

### Verification Steps

1. Cypress `block.spec.js` should pass
Steps - 
Go to venia-integration-test dir
Run `yarn install`
Run `yarn test`
Run `block.spec.js` from cypress ui

## Checklist

* I have added tests to cover my changes, if necessary.
* I have added translations for new strings, if necessary.
* I have updated the documentation accordingly, if necessary.

### CODE REVIEW

**Review by larsroettig — DISMISSED** [review]
LGTM

**Comment by michaelyu0123** [comment]
> @michaelyu0123 do you know why ``yarn build` don't work should we takle in QA

It is in the process of getting resolved in https://github.com/magento/pwa-studio/pull/3329
Could probably just wait till EOD Monday otherwise just move forward.

**Review by larsroettig — DISMISSED** [review]

File: venia-integration-tests/src/fixtures/pageBuilder/block/cmsBlock.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
...\" anchor_text=\"\" id_path=\"\" show_pager=\"0\" products_count=\"2\" condition_option=\"condition\" condition_option_value=\"\" type_name=\"Catalog Products List\" conditions_encoded=\"^[`1`:^[`type`:`Magento||CatalogWidget||Model||Rule||Condition||Combine`,`aggregator`:`all`,`value`:`1`,`new_child`:``^],`1--1`:^[`type`:`Magento||CatalogWidget||Model||Rule||Condition||Product`,`attribute`:`category_ids`,`operator`:`==`,`value`:`3`^]^]\" sort_order=\"name_descending\"}}</div></div><div data-content-type=\"row\" data-appearance=\"contained\" data-element=\"main\" style=\"display: none;\"><div data-enable-parallax=\"0\" data-parallax-speed=\"0.5\" data-background-images=\"{}\" data-element=\"inner\" style=\"justify-content: flex-start; display: flex; flex-direction: column; background-position: left top; background-size: cover; background-repeat: no-repeat; background-attachment: scroll; border-style: none; border-width: 1px; border-radius: 0px; margin: 0px 0px 10px; padding: 10px;\"><h2 data-content-type=\"heading\" data-appearance=\"default\" data-element=\"main\" style=\"text-align: center; border-style: none; border-width: 1px; border-radius: 0px;\">Hidden Row</h2></div></div>",
```
Comment (tjwiebell) [inline venia-integration-tests/src/fixtures/pageBuilder/block/cmsBlock.json:5]: The `block.json` looks updated, but the remaining three fixtures appear to have old PB content data with inline styles. Do these three fixtures need updated?

Comment (michaelyu0123) [reply venia-integration-tests/src/fixtures/pageBuilder/block/cmsBlock.json]: The fixtures have been updated.

**Review by tjwiebell — APPROVED** [review]
I updated snaps with correct viewport, re-approving.

**Comment by tjwiebell** [comment]
Ran PageBuilder suite locally, all passing now.

```
       Spec                                              Tests  Passing  Failing  Pending  Skipped  
  ┌────────────────────────────────────────────────────────────────────────────────────────────────┐
  │ ✔  integrationTests/pageBuilder/banner      02:48       10       10        -        -        - │
  │    .spec.js                                                                                    │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  integrationTests/pageBuilder/block.      01:01        4        4        -        -        - │
  │    spec.js                                                                                     │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  integrationTests/pageBuilder/button      00:25        1        1        -        -        - │
  │    s.spec.js                                                                                   │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  integrationTests/pageBuilder/column      00:25        1        1        -        -        - │
  │    .spec.js                                                                                    │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  integrationTests/pageBuilder/divide      00:15        1        1        -        -        - │
  │    r.spec.js                                                                                   │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  integrationTests/pageBuilder/dynami      00:09        1        1        -        -        - │
  │    cBlock.spec.js                                                                              │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  integrationTests/pageBuilder/image.      00:37        1        1        -        -        - │
  │    spec.js                                                                                     │
  ├────────────────────────────────────────────────────────────────────────────────────────────────┤
  │ ✔  integrationTests/pageBuilder/text.s      00:29        1        1        -        -        - │
  │    pec.js                                                                                      │
  └────────────────────────────────────────────────────────────────────────────────────────────────┘
    ✔  All specs passed!                        06:12       20       20        -        -        -  
```

**Pull request merged**